dist/front-phone/4.14.0/phone-all-bundle.js

```diff
@@ -0,0 +1,3 @@
+/*! front-phone - v4.13.1 - https://vtex.github.io/front.phone/ */
```

Comment:
Could you create this build files again but after the tag front.phone tag is created?